docs/topics/downloader-middleware.rst

```diff
@@ -474,7 +474,7 @@ DBM storage backend
 
     A DBM_ storage backend is also available for the HTTP cache middleware.
 
-    By default, it uses the anydbm_ module, but you can change it with the
+    By default, it uses the `dbm module`_, but you can change it with the
```

Comment:
Since we have InterSphinx configured for Python documentation, you should be able to use:

```suggestion
    By default, it uses the :mod:`dbm`, but you can change it with the
```

And not need the link reference from line 1205 anymore.